src/components/molecules/card/card.scss

```
@@ -0,0 +1,49 @@
+@use "../../variables" as *;
+/*BootstrapのCardを使う*/
+/*基本縦並びで、breakpointが来たら横並びになる、縦長な感じ*/
+.card {
+  border: none;
+  & > .card-upper {
+    overflow: hidden;
+    & > img {
+      width: 100%;
+      transition: 0.5s;
+      transition-timing-function: ease;
```

> transition は [CSS](https://developer.mozilla.org/ja/docs/Web/CSS) のプロパティで、 [transition-property](https://developer.mozilla.org/ja/docs/Web/CSS/transition-property)、 [transition-duration](https://developer.mozilla.org/ja/docs/Web/CSS/transition-duration)、 [transition-timing-function](https://developer.mozilla.org/ja/docs/Web/CSS/transition-timing-function)、 [transition-delay](https://developer.mozilla.org/ja/docs/Web/CSS/transition-delay) の[一括指定プロパティ](https://developer.mozilla.org/ja/docs/Web/CSS/Shorthand_properties)です。


https://developer.mozilla.org/ja/docs/Web/CSS/transition

とのことなので、
```
transition: 0.5s;
```
と書いた場合
```
transition-duration: 0.5s;
```
として解釈されています。
transitionプロパティを使うならtransition-timing-functionもtransitionの中に含めてしまった方がいいと思います。（ちなみにtransition-functionは初期値がeaseなので指定しなくても大丈夫です。）
```suggestion
      transition: 0.5s ease;
```

---

なるほど！border: 1px solid $black; と並列するのと同じ感覚ですね！
良く分かりました、ありがとうございます！

---

まさにそんな感じです！！